neat_core.c

```diff
@@ -30,6 +33,9 @@
 #endif
 #ifdef __linux__
     #include "neat_linux_internal.h"
+#if !defined(HAVE_NETINET_SCTP_H) && !defined(USRSCTP_SUPPORT)
+    #include <linux/sctp.h>
```

Comment:
I find this also strange...
I think you can neat without having libsctp-dev installed, SCTP wouldn't just work. So it is only a requirement when you want to use SCTP. But we can just require it and therefore adding it to the required libraries for Linux.